src/SixLabors.Fonts/RendererOptions.cs

```diff
@@ -71,11 +73,81 @@ public RendererOptions(Font font, float dpi, Vector2 origin)
+        /// <param name="dpi">The dpi.</param>
+        /// <param name="fallbackFontFamilies">An optional collection of font families to fallback to if the glyph is missing from the <paramref name="font"/>.</param>
+        public RendererOptions(Font font, float dpi, IEnumerable<FontFamily> fallbackFontFamilies)
+            : this(font, dpi, dpi, fallbackFontFamilies)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="RendererOptions"/> class.
+        /// </summary>
+        /// <param name="font">The font.</param>
+        /// <param name="dpiX">The X dpi.</param>
+        /// <param name="dpiY">The Y dpi.</param>
+        /// <param name="fallbackFontFamilies">An optional collection of font families to fallback to if the glyph is missing from the <paramref name="font"/>.</param>
+        public RendererOptions(Font font, float dpiX, float dpiY, IEnumerable<FontFamily> fallbackFontFamilies)
+            : this(font, dpiX, dpiY, Vector2.Zero, fallbackFontFamilies)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="RendererOptions"/> class.
+        /// </summary>
+        /// <param name="font">The font.</param>
+        /// <param name="origin">The origin location.</param>
+        /// <param name="fallbackFontFamilies">An optional collection of font families to fallback to if the glyph is missing from the <paramref name="font"/>.</param>
+        public RendererOptions(Font font, Vector2 origin, IEnumerable<FontFamily> fallbackFontFamilies)
+            : this(font, 72, 72, origin, fallbackFontFamilies)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="RendererOptions"/> class.
+        /// </summary>
+        /// <param name="font">The font.</param>
+        /// <param name="dpi">The dpi.</param>
+        /// <param name="origin">The origin location.</param>
+        /// <param name="fallbackFontFamilies">An optional collection of font families to fallback to if the glyph is missing from the <paramref name="font"/>.</param>
+        public RendererOptions(Font font, float dpi, Vector2 origin, IEnumerable<FontFamily> fallbackFontFamilies)
+            : this(font, dpi, dpi, origin, fallbackFontFamilies)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="RendererOptions"/> class.
+        /// </summary>
+        /// <param name="font">The font.</param>
+        /// <param name="dpiX">The X dpi.</param>
+        /// <param name="dpiY">The Y dpi.</param>
+        /// <param name="origin">The origin location.</param>
+        /// <param name="fallbackFontFamilies">An optional collection of font families to fallback to if the glyph is missing from the <paramref name="font"/>.</param>
+        public RendererOptions(Font font, float dpiX, float dpiY, Vector2 origin, IEnumerable<FontFamily> fallbackFontFamilies)
```

Comment:
I think you're much better removing all these constructor overloads and using default property assignment.

Reply:
hmm... nether `Vector2` or `IEnumerable<FontFamily>` can be default parameters especially as we won't want to enable c#8 nullable referenece types and don't want to expose the arguments as nullable... I will however remove the fallback arguments and move them over to a setable property.

Reply:
Couldn't you use `Vector2?` and `IReadOnlyCollection<T>` for the properties?

We use `PointF?` in `ResizOptions`.

Reply:
1. we could do but we shouldn't be encouraging magic null values in my opinion and 2. I would rather not break the API at this stage without good reason, these constructors are already in use by imagesharp and other third parties.